### PR TITLE
SW-2083 / SW-2082 : fix accession number display + remove ready-by-date from withdrawal form

### PR DESCRIPTION
**Pull request by karthikbtf**
- updated search field to pull in accession number
- fixed withdraw form to show correct content

Terraware App 2022-11-01 12-44-53

Terraware App 2022-11-01 13-52-37

Terraware App 2022-11-01 13-53-03

### CODE REVIEW

File: src/components/Inventory/view/WithdrawalModal.tsx [hunk 1]
```diff
@@ -259,82 +259,52 @@ export default function WithdrawalsModal(props: WithdrawalsModalProps): JSX.Elem
...
-                          onChange={onChange}
-                          type='text'
-                          label={strings.WITHDRAW_QUANTITY}
-                          display={true}
-                        />
-                      </Grid>
-                    </>
-                  );
-                })}
+                {record.batchWithdrawals.map((bw, index) => (
```
Comment (karthikbtf) [inline src/components/Inventory/view/WithdrawalModal.tsx:262]: i mostly moved grid content up to fill previous deleted grid-content (Ready by date).
Also fixed a non-unique key error (introduced Grid container)

File: src/components/Inventory/view/WithdrawalModal.tsx [hunk 2]
```diff
@@ -259,82 +259,52 @@ export default function WithdrawalsModal(props: WithdrawalsModalProps): JSX.Elem
...
+                        value={record.withdrawnDate}
+                        onChange={changeDate}
+                        errorText={validateFields && !record.withdrawnDate ? strings.REQUIRED_FIELD : ''}
+                      />
+                    </Grid>
+                  </Grid>
+                ))}
               </>
             )}
-            <Grid
```
Comment (constanzauanini) [inline src/components/Inventory/view/WithdrawalModal.tsx:322]: We need this outside the map({ ... } to show it on 'Out plant' purpose

Comment (karthikbtf) [reply src/components/Inventory/view/WithdrawalModal.tsx:322]: hmmm ok, good catch, thanks. will fix.

**Review by constanzauanini — APPROVED** [review]

**Review by constanzauanini — CHANGES_REQUESTED** [review]
I think estimated date is gone now :(

**Pull request merged**